### ISSUE TITLE
Enhanced BLN by @SultanXDA

### ISSUE THREAD

**Issue by javelinanddart** [Grarak/KernelAdiutor#186]
On/off Switch: /sys/devices/virtual/misc/enhanced_bln/blink_control (1 is on, 0 is off)
Time (in milliseconds) before keeping backlights permanently on to conserve power:
/sys/devices/virtual/misc/enhanced_bln/blink_timeout_ms (if set to zero, backlights will be always on if a notification comes, no max value, unless we test controlled battery drain)
sys/devices/virtual/misc/enhanced_bln/blink_interval_ms
Two values in this file
First one is the time to keep the backlight on while it's blinking, second one is the time to keep it off
Basically pulse duration
Developers Description:
First Value: 
@on_ms:     Frequency in milliseconds to keep the backlights on while
            blinking them. Controlled by userspace

Second Value: 
@off_ms:        Frequency in milliseconds to keep the backlights off while
            blinking them. Controlled by userspace.

**Comment by javelinanddart** [comment]
Bump
